### DISCUSSION TITLE
Point atributte edits

### DISCUSSION THREAD

**Discussion by Vladnita87** [opengisch/QField#4522] [Q&A]
Hello gents,

I have a point layer with predefined XY and Z=0 coordonates.
I will add this to my qfield project.
The qfield tablet i connect to external GNSS antenna, corrections using NTRIP.

Now my question is how i can just modify the XYZ coordonates of the point in the measured location.Example i have a point in a bad location and i want to move it to other position...so it is like modify point features, so i want in the predefined x y z field to record the measured coordonates.
Also it is possible using qfield cloud to sync multiple devices measurments based on this layer?

Thanks

**Comment by Bernardodufutur65** [comment]
Hi,

I have the same kind of project but i dont want to move the point for me. 
My project is to create a point using GNSS and calculate the differences between the new point and the oldest one. So if you have any news or answers about it, please tell me. 

For your next question, I think you can simply push your new work on Qfield and then the other devices need to sync their project.
